lib/charliecloud.py

```diff
@@ -210,12 +220,36 @@ def __init__(self, ref, download_cache, unpack_dir, image_subdir=None):
    def __str__(self):
       return str(self.ref)
 
+   @property
+   def default_arch_dlcache(self):
+      return os.path.dirname(os.path.dirname(
+                             self.download_cache)) + '/amd64/dlcache'
```

Comment:
Should we change all these path properties (including the existing ones) to start with `path_`, in keeping with the most-significant-first naming convention?